src/Libraries/PythonNodeModelsWpf/IronPythonCompletionData.cs

```diff
@@ -15,7 +15,6 @@ public class IronPythonCompletionData : ICompletionData
     {
```

Comment:
Is this class also used for autocomplete in PythonNet?

Reply:
no, it's used to decouple AvalonEdit's interface (and WPF deps) from the core completion types. We can think of this as a UI type, thats why it's in this PythonWPF assembly.